docs/streams/developer-guide/dsl-api.html

```diff
@@ -3738,7 +3738,7 @@ <h5><a class="toc-backref" href="#id34">KTable-KTable Foreign-Key
             </div>
         </div>
         <div class="section" id="testing-a-streams-app">
-            <a class="headerlink" href="#testing-a-streams-app" title="Permalink to this headline"><h2>Testing a Streams application</h2></a>
+            <h2><a class="headerlink" href="#testing-a-streams-app" title="Permalink to this headline">Testing a Streams application</a></h2>
```

Comment:
Currently, the `TESTING A STREAMS APPLICATION` header is in blue, which is different from others. Turns out that it's because we put `<h2>` tag inside `<a>`, and cause to apply the wrong css format.
![image](https://user-images.githubusercontent.com/43372967/81927117-b1d18780-9615-11ea-979b-db48c294e742.png)